### PR TITLE
A gigabyte is 1024 instead of 1000

### PR DESCRIPTION
**Pull request by brittag**
Based on how the platform actually works. Includes updates for rates page as well as spreadsheets.

### CODE REVIEW

File: content/overview/pricing/rates.md [hunk 1]
```diff
@@ -72,11 +72,11 @@ Free of charge and without any paperwork, anyone with a government email address
...
 ## Quota pricing
 
-Quotas are priced at $0.0033 per MB reserved per day and assessed monthly. In practice, this works out to $99 per GB per month, based on a 30-day month:
+Quotas are priced at $0.0033 per MB reserved per day and assessed monthly. In practice, this works out to about $100 per GB per month:
 
-`$0.0033 * 1000 MB * 30 days = $99 per GB per month`
+`$0.0033 * 1024 MB * 30 days = $101.37 per GB per month`
 
-The monthly cost will vary slightly from $92.40 per GB in a 28-day month to $102.30 per GB in a 31-day month.
+The monthly cost will vary slightly from $94.62 per GB in a 28-day month to $104.76 per GB in a 31-day month.
```
Comment (khandelwal) [inline content/overview/pricing/rates.md:79]: Do we really need this clarification or detail here? I think it adds more text to a page that's already wordy.

File: content/overview/pricing/rates.md [hunk 2]
```diff
@@ -33,31 +33,31 @@ You can email [cloud-gov-inquiries@gsa.gov](mailto:cloud-gov-inquiries@gsa.gov)
 The [prototyping package]({{< relref "overview/pricing/prototyping.md" >}}) lets many teams build and deploy apps, with full access control. Prototyping packages cannot host production data; all apps are limited to the `*.app.cloud.gov` domain. Whole agencies or departments often purchase a single prototyping package.
 
 **Annual fee:** $15K<br>
-**Resource usage quota:** ~$99 per GB per month[*](#quota-pricing), up to any limit set by user<br>
+**Resource usage quota:** ~$100 per GB per month[*](#quota-pricing), up to any limit set by user<br>
```
Comment (khandelwal) [inline content/overview/pricing/rates.md:36]: Our price for usage is always the same regardless of package. Do we need to repeat it in every description? Is there a way to make this more compact? 

The limits also do not change, except for Open Data.

File: content/overview/pricing/rates.md (hunk 1 above)
Comment (khandelwal) [reply content/overview/pricing/rates.md:79]: I do like the change from 1000MB to 1024MB.

File: content/overview/pricing/rates.md [hunk 3]
```diff
@@ -33,31 +33,31 @@ You can email [cloud-gov-inquiries@gsa.gov](mailto:cloud-gov-inquiries@gsa.gov)
...
 ### Open data
 
 Host one public-facing system with no confidentiality risk assessed. Open data packages include all the spaces you need and DNS support. This is the lightest production-level package.
 
 **Annual fee:** $10K<br>
-**Resource usage quota:** ~$99 per GB per month[*](#quota-pricing), up to 2GB<br>
+**Resource usage quota:** ~$100 per GB per month[*](#quota-pricing), up to 2GB<br>
 **Managed services:** Full access
 
 ### FISMA Low
```
Comment (khandelwal) [inline content/overview/pricing/rates.md:47]: While we're editing this page, is there something we can add for indicating that these include dev, staging, production (or whatever configuration you desire)? I feel like we get that question often enough.

File: content/overview/pricing/rates.md [hunk 4]
```diff
@@ -33,31 +33,31 @@ You can email [cloud-gov-inquiries@gsa.gov](mailto:cloud-gov-inquiries@gsa.gov)
 The [prototyping package]({{< relref "overview/pricing/prototyping.md" >}}) lets many teams build and deploy apps, with full access control. Prototyping packages cannot host production data; all apps are limited to the `*.app.cloud.gov` domain. Whole agencies or departments often purchase a single prototyping package.
 
 **Annual fee:** $15K<br>
-**Resource usage quota:** ~$99 per GB per month[*](#quota-pricing), up to any limit set by user<br>
+**Resource usage quota:** ~$100 per GB per month[*](#quota-pricing), up to any limit set by user<br>
 **Managed services:** Full access
 
 ### Open data
 
 Host one public-facing system with no confidentiality risk assessed. Open data packages include all the spaces you need and DNS support. This is the lightest production-level package.
```
Comment (khandelwal) [inline content/overview/pricing/rates.md:41]: I don't know that someone looking at this page (one can imagine that pricing is one of the first few links you look at) is going to know what a 'space' is.

**Review by khandelwal — CHANGES_REQUESTED** [review]

**Comment by brittag** [comment]
@khandelwal All of these are good points that will take more work and that we should address in another PR, so that we can get this one merged since it fixes a current problem. :)

**Comment by khandelwal** [comment]
Fair enough. Let's get that other PR started soon.

**Review by khandelwal — APPROVED** [review]

**Pull request merged**